test/service_instance_and_binding_test/service_binding_test/service_binding_test.go

```diff
@@ -80,10 +80,14 @@ func (t *testCase) responseByBrokerOrClientMode(expected int, statusByBrokerResp
 }
 
 var _ = DescribeTestsFor(TestCase{
-	API: web.ServiceBindingsURL,
+	API:                             web.ServiceBindingsURL,
```

Comment:
we should test all possible rest actions on reference plan instances not just post (patch,delete, get)

Reply:
we have:
post binding
* reference instance
* shared instance
delete binding
* reference instance
* shared instance
get binding
* reference instance
* shared instance
get parameters
* reference instance
* shared instance